### PR TITLE
Jroach/us127885

### PR DESCRIPTION
**Pull request by jeroach**
[US127885](https://rally1.rallydev.com/#/42960320374d/dashboard?detail=%2Fuserstory%2F603249278956&fdp=true?fdp=true): [Consistent Eval] Published assessment warning is not appearing in rubric pop out in CE pages

There are a _lot_ of files changed - however the vast majority are simply rubric demo/test data that was copied directly from the `d2l-rubric` repo, in order to fix our rubric demo and vdiff tests.

Feel free to ignore all files under:
`/demo/data/rubric/`
`/test/perceptual/data/rubric`

### CODE REVIEW

File: components/right-panel/consistent-evaluation-popup-rubric.js [hunk 1]
```diff
@@ -32,6 +44,13 @@ export class ConsistentEvaluationPopupRubric extends LocalizeConsistentEvaluatio
 			return html ``;
 		}
 		return html`
+			<d2l-alert
```
Comment (jeroach) [inline components/right-panel/consistent-evaluation-popup-rubric.js:47]: The bulk of the logic for the US is in this file

File: karma.sauce.conf.js [hunk 2]
```diff
@@ -13,11 +13,6 @@ const customLaunchers = {
 		browserName: 'firefox',
 		platform: 'macOS 11.00'
 	},
-	safari: {
```
Comment (jeroach) [inline karma.sauce.conf.js:16]: Safari tests were failing repeatedly (timeouts) on Sauce CI - disabling for now.

If anyone has a better solution in mind I'm all ears.

File: test/perceptual/consistent-evaluation-popup-rubric.visual-diff.html [hunk 3]
```diff
@@ -40,6 +49,13 @@
 			></d2l-consistent-evaluation-popup-rubric>
 		</div>
 
+		<div id="published-warning">
```
Comment (jeroach) [inline test/perceptual/consistent-evaluation-popup-rubric.visual-diff.html:52]: New vdiff test for the feature

File: test/perceptual/consistent-evaluation-popup-rubric.visual-diff.js [hunk 4]
```diff
@@ -19,12 +19,17 @@ describe('d2l-consistent-evaluation-popup-rubric', () => {
 
 	after(async() => await browser.close());
 
-	it.skip('renders rubric popout', async function() {
+	it('renders rubric popout', async function() {
```
Comment (jeroach) [inline test/perceptual/consistent-evaluation-popup-rubric.visual-diff.js:22]: These have been fixed

File: test/perceptual/consistent-evaluation-rubric.visual-diff.js [hunk 5]
```diff
@@ -19,17 +19,17 @@ describe('d2l-consistent-evaluation-rubric', () => {
 
 	after(async() => await browser.close());
 
-	it.skip('renders rubric', async function() {
+	it('renders rubric', async function() {
```
Comment (jeroach) [inline test/perceptual/consistent-evaluation-rubric.visual-diff.js:22]: These have been fixed

**Review by amauryperez — APPROVED** [review]
Looks like a pretty simple change to me. it sucks having spent so much time with the demos and tests  :-(

**Review by JamieTran — APPROVED** [review]

**Pull request merged**